### ISSUE TITLE
Move documentation outside the sources

### ISSUE THREAD

**Issue by juherr** [cbeust/testng#635]
What do you think to move the documentation outside the sources?

I see 2 options that will keep the community spirit:
- Convert html pages into Mardown and move them into the wiki, and/or
- Use the GitHub Pages feature.

**Comment by FibreFoX** [comment]
+1 for markdown, even without having the project on github its worth doing this (makes it easier to extend/rework)

**Comment by cbeust** [comment]
Not a big fan of this for a couple of reasons:

- The current documentation has some special styling and Javascript code (for the highlighting, the style and the banner)
- I'd rather not tie the documentation to an external web site (github). Imagine if I had done that when TestNG was on Google Code...

**Comment by juherr** [comment]
Github Pages doesn't have the drawbacks you described: you just need a specific branch (`gh-pages`) and a CNAME file.
I did the work to show you on [my fork](https://github.com/juherr/testng/tree/gh-pages) with [my custom domain](http://testng.juherr.herr.fr/).
Nothing changed. So it can be a first step.

I converted a chapter of the doc into a Mardown wiki page too: https://github.com/juherr/testng/wiki/testng.xml
I agree we are currently lossing the look&field but I'm not an expert of Mardown and I saw AcsiiDoc is also possible (and I know ad is more powerfull).
